### PR TITLE
fix: no --tag prerelease for private module

### PR DESCRIPTION
**Pull request by whyvez**
fixes: #71

### CODE REVIEW

**Comment by stevemao** [comment]
Thanks for the PR @whyvez , I think your "fixes" issue number is wrong.

File: test.js [hunk 1]
```diff
@@ -440,6 +440,14 @@ describe('cli', function () {
       commit('feat: first commit')
       execCli('--prerelease alpha').stdout.should.include('--tag alpha')
     })
+
+    it('does not advises use of --tag prerelease for private modules', function () {
```
Comment (stevemao) [inline test.js]: "does not advise"

**Comment by whyvez** [comment]
preceded by https://github.com/conventional-changelog/standard-version/pull/296

**Comment by stevemao** [comment]
You didn't have to open a new one since you can just force push.

**Comment by whyvez** [comment]
@stevemao The branch name is wrong too though. 😬

**Pull request closed without merging**